### PR TITLE
Translate for last part of transfer learning tutorial

### PR DESCRIPTION
**Pull request by jafffy**
## 라이선스 동의
_변경해주시는 내용에 BSD 3항 라이선스가 적용됨을 동의해주셔야 합니다._<br />
_더 자세한 내용은 [기여하기 문서](https://github.com/9bow/PyTorch-tutorials-kr/blob/master/CONTRIBUTING.md)를 참고해주세요._<br />
_동의하시면 아래 `[ ]`를 `[x]`로 만들어주세요._<br />

- [x] 기여하기 문서를 확인하였으며, 본 PR 내용에 BSD 3항 라이선스가 적용됨에 동의합니다.

## 관련 이슈 번호
_이 Pull Request와 관련있는 이슈 번호를 적어주세요._<br />
_이슈 또는 PR 번호 앞에 #을 붙이시면 제목을 바로 확인하실 수 있습니다. (예. #999 )_

- **이슈 번호**: #(숫자)

## PR 종류
_이 PR에 해당되는 종류 앞의 `[ ]`을 `[x]`로 변경해주세요._<br />
- [ ] 오탈자를 수정하거나 번역을 개선하는 기여
- [x] 번역되지 않은 튜토리얼을 번역하는 기여
- [ ] 공식 튜토리얼 내용을 반영하는 기여
- [ ] 위 종류에 포함되지 않는 기여

## PR 설명
_이 PR로 무엇이 달라지는지 대략적으로 알려주세요._

Transfer learning 의 일부를 번역하였습니다.

### CODE REVIEW

File: beginner_source/transfer_learning_tutorial.py [hunk 1]
```diff
@@ -284,44 +280,43 @@ def visualize_model(model, num_images=6):
...
-# ConvNet as fixed feature extractor
+# 고정된 특정 추출기로서의 ConvNet
 # ----------------------------------
 #
-# Here, we need to freeze all the network except the final layer. We need
-# to set ``requires_grad == False`` to freeze the parameters so that the
-# gradients are not computed in ``backward()``.
+# 이번에는 마지막 레이어 외의 모든 레이어들을 얼려봅시다. 얼려진 레이어들의 파라미터들은 ``backward()`` 함수가
+# 호출되더라도 해당 레이어들의 그래디언트는 계산되지 않습니다. 특정 파라미터를 얼리기 위해서는 ``requires_grad == False``를
+# 호출하면 됩니다.
```
Comment (hrxorxm) [inline beginner_source/transfer_learning_tutorial.py:288]: 어떤 표현이 더 자주 쓰이는지는 모르겠지만, freeze를 "얼린다"기보다 레이어를 "고정한다"고 표현하면 어떨까요?

**Review by hrxorxm — COMMENTED** [review]
전체적으로 읽기 쉽게 잘 번역해주신 것 같습니다!

File: beginner_source/transfer_learning_tutorial.py [hunk 2]
```diff
@@ -244,34 +241,33 @@ def visualize_model(model, num_images=6):
         model.train(mode=was_training)
 
 ######################################################################
-# Finetuning the convnet
+# convnet 파인 튜닝 하기
 # ----------------------
 #
-# Load a pretrained model and reset final fully connected layer.
+# 미리 학습된 (pretrained) 모델을 로딩한 후, 마지막 FC 레이어만 커스터마이즈합니다.
```
Comment (miNept) [inline beginner_source/transfer_learning_tutorial.py:247]: Fully connected Layer를 FC레이어로 축약하는 것보다, 한국어로 번역 후 괄호로 설명을 해주면 더 이해하기 쉬울 것 같습니다!

File: beginner_source/transfer_learning_tutorial.py [hunk 3]
```diff
@@ -151,37 +148,37 @@ def train_model(model, criterion, optimizer, scheduler, num_epochs=25):
...
             running_corrects = 0
 
-            # Iterate over data.
+            # 모든 데이터에 대해 반복합니다.
             for inputs, labels in dataloaders[phase]:
                 inputs = inputs.to(device)
                 labels = labels.to(device)
 
-                # zero the parameter gradients
+                # 파라미터 그래디언트를 0으로 설정
```
Comment (miNept) [inline beginner_source/transfer_learning_tutorial.py:166]: 용어집에서 gradient를 "변화도"로 번역하고 있는 것 같습니다!

File: beginner_source/transfer_learning_tutorial.py (hunk 1 above)
Comment (miNept) [reply beginner_source/transfer_learning_tutorial.py:288]: 저도 freeze를 "고정한다"가 더 이해하기에 쉬울 것 같습니다

**Review by miNept — COMMENTED** [review]
몇가지 축약어를 조금만 더 쉽게 풀어주시면 좋을 것 같습니다..!

File: beginner_source/transfer_learning_tutorial.py [hunk 4]
```diff
@@ -151,37 +148,37 @@ def train_model(model, criterion, optimizer, scheduler, num_epochs=25):
...
-                # track history if only in train
+                # 순전파
+                # 학습 단계일 때는 평가를 위한 이력을 추적합니다.
                 with torch.set_grad_enabled(phase == 'train'):
                     outputs = model(inputs)
                     _, preds = torch.max(outputs, 1)
                     loss = criterion(outputs, labels)
 
-                    # backward + optimize only if in training phase
+                    # 학습 단계일 때는 역전파 및 최적화 수행합니다.
```
Comment (Jo-ad-fontes) [inline beginner_source/transfer_learning_tutorial.py:176]: '최적화 -> 최적화를'로 조사가 빠진 것 같아요!

**Review by Jo-ad-fontes — COMMENTED** [review]
좋은 번역 감사합니다~

File: beginner_source/transfer_learning_tutorial.py [hunk 5]
```diff
@@ -128,19 +128,16 @@ def imshow(inp, title=None):
...
+# 모델 학습하기
 # ------------------
 #
-# Now, let's write a general function to train a model. Here, we will
-# illustrate:
+# 모델 학습을 위한 함수를 작성해 보겠습니다. 본 함수에서는 아래와 같은 기능을 수행합니다.
 #
-# -  Scheduling the learning rate
-# -  Saving the best model
+# - learning rate 를 스케쥴링
```
Comment (hyoyoung) [inline beginner_source/transfer_learning_tutorial.py:136]: learning rate가 용어집에 있습니다 참고 바랍니다

File: beginner_source/transfer_learning_tutorial.py [hunk 6]
```diff
@@ -128,19 +128,16 @@ def imshow(inp, title=None):
...
+# - 최적의 모델을 저장
+#
+# 아래의 코드에서 ``scheduler`` 라는 인자는 ``torch.optim.lr_scheduler`` 에 속하는 LR scheduler 오브젝트입니다.
```
Comment (hyoyoung) [inline beginner_source/transfer_learning_tutorial.py:139]: 위에서 스케쥴이라고 썼으므로 scheduler 오브젝트는 스케쥴러 객체로 바꾸는게 좋지 않을까요?

File: beginner_source/transfer_learning_tutorial.py [hunk 7]
```diff
@@ -205,16 +202,16 @@ def train_model(model, criterion, optimizer, scheduler, num_epochs=25):
         time_elapsed // 60, time_elapsed % 60))
     print('Best val Acc: {:4f}'.format(best_acc))
 
-    # load best model weights
+    # 최적 모델 가중치 로딩
```
Comment (hyoyoung) [inline beginner_source/transfer_learning_tutorial.py:205]: 로딩보다는 불러오기가 자연스러울 듯 합니다

**Review by hyoyoung — CHANGES_REQUESTED** [review]
